lib/appdmg.js

```diff
@@ -80,6 +80,14 @@ module.exports = exports = function (options) {
     fs.writeFile(global.target, '', { flag: 'wx' }, function (err) {
       if (err && err.code === 'EEXIST') return next(new Error('Target already exists'))
 
+      pipeline.addCleanupStep('unlink-target', 'Removing target image', function (next, hasErrored) {
```

Comment:
`err` could potentially be non-null here in which case the file won't exists. Maybe add `if (err) return next(err)` before?

Reply:
actually, it might be that if `fs.writeFile` returns an `err`, we do not know if the file is there or not. In that case we should always unlink it, but ignore any `ENOENT` errors...

Not super important to get this done now though, if you're short on time 👍